openmls/src/tree/sender_ratchet.rs

```diff
@@ -11,8 +11,47 @@ use crate::tree::{index::LeafIndex, secret_tree::*};
 use super::index::NodeIndex;
 use super::*;
 
-const OUT_OF_ORDER_TOLERANCE: u32 = 5;
-const MAXIMUM_FORWARD_DISTANCE: u32 = 1000;
+// TODO #265: This should disappear
+pub(crate) const OUT_OF_ORDER_TOLERANCE: u32 = 5;
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct SenderRatchetConfiguration {
```

Comment:
I think this should get some documentation as well. Maybe just summarize or copy some of the documentation from `new()`.